init_files/v1/iplant_backup.sh

```diff
@@ -1,346 +0,0 @@
-#!/bin/bash
-
-################################################################
-#
-#  Script to backup or retrieve data between an Atmosphere
-#  instance and iPlant Datastore
-#  
-#  iplant_backup.sh -- Version 1.0
```

Comment:
According to the [Atmosphere(0) user manual](https://wiki.cyverse.org/wiki/display/atmman/Backing+Up+and+Restoring+Your+Data+to+the+Data+Store) there is an `iplant_backup` binary in the $PATH of all Atmosphere instances, and looking at a random live instance, this seems to be true. (This file is also stored in atmosphere-ansible's [atmo-idrop](https://github.com/cyverse/atmosphere-ansible/blob/c49d4fb25bf1e0881ef694608e3b49b3ae654c27/ansible/roles/atmo-idrop/files/iplant_backup.sh) role, although that doesn't appear to actually install it the script on each new instance, so I'm unsure how it gets there.) Regardless, I think that's something CyVerse-specific and doesn't belong in this repo.